### ISSUE TITLE
Maniac Patch: Animation Type "Step Frame Fix"

### ISSUE THREAD

**Issue by TitanKaempfer** [EasyRPG/Player#3277]
**Name of the game:** EasyRPG-ManiacPatchGraphics-Example
**Player platform:** Windows, continuous build of EasyRPG Player
**Issue:**
If an event is placed with the animation type of "Step Frame Fix" and a graphic which's pattern is set to either left or right, it will be shown exactly as selected in an RPG_RT.exe patched with the Maniac Patch.
However if the same game is played with the current EasyRPG player, the fixed step frame will snap to the middle pattern frame, which may or may not break games visually.

Example game and screenshots are attached.

**Screenshots:**
![ManiacPatch](https://github.com/user-attachments/assets/599adc1d-379a-40af-94cc-447bc859bcc0)
How it looks like in the RPG_RT.exe with Maniac Patch applied.

![EasyRPG](https://github.com/user-attachments/assets/9d630e4c-fbee-4201-bd84-094d26d77d09) 
How it looks like in EasyRPG player.

**Download to the example game:**
[EasyRPG-ManiacPatchGraphics-Example.zip](https://github.com/user-attachments/files/17180842/EasyRPG-ManiacPatchGraphics-Example.zip)

**Comment by Mimigris** [comment]
Is it related to the Maniac patch? Step Frame Fix is a feature from the English RPG Maker 2003, not Maniac, so this needs to be known if it's tied to a difference with the patch.

**Comment by Ghabry** [comment]
Yes this is a 2k3e feature (actually this is already supported in older versions of the engine. Cherry only added a way to select it in the editor).

**Comment by TitanKaempfer** [comment]
It is included in Cherrys version of RM2K3 yes. But the behaviour is slightly changed.
An unpatched game results in the same behaviour as EasyRPG's player is currently handling it. Maniac Patch changes the behaviour to function as the screenshot shown above.
